python/paddle/nn/layer/common.py

```diff
@@ -30,6 +30,52 @@ def _npairs(x, n):
     return x
 
 
+class Identity(Layer):
+    r"""
+
+    A placeholder identity operator that is argument-insensitive. For each input :math:`X` ,
+    the output :math:`Out` is:
+
+    .. math::
+
+        Out = X
+
+    Parameters:
+        Any parameters which is not used
+
+    Shape:
+        - input: Multi-dimentional tensor with shape :math:`[batch\_size, n1, n2, ...]` .
+        - output: Multi-dimentional tensor with shape :math:`[batch\_size, n1, n2, ...]` .
+
+    Examples:
+        .. code-block:: python
+
+          import paddle
+          layer = paddle.nn.Identity(54)
+          input_tensor = paddle.to_tensor(paddle.randn(shape=[3, 2]))
+          input_tensor.stop_gradient=False
+          input_tensor = input_tensor+1
+          input_tensor.register_hook(lambda grad: print('input grad', grad))
+          print('input_tensor.grad', input_tensor.grad)
+          out = m(input_tensor)
```

Comment:
`out = m(input_tensor)` -> `out = layer(input_tensor)`
You can delete the calculation of gradient in the example.